### DISCUSSION TITLE
Do you support medical image processing?

### DISCUSSION THREAD

**Discussion by Enderfga** [autogluon/autogluon#1771] [Q&A] [answered]
Hello, I am recently working on a classification task for medical images, the images are in NIFTI format (with nii suffix) and would like to ask if I can use autogluon to help me with this work？

**Comment by Qiaochu-Song** [comment, marked as answer]
Hi, I am a user too and i think this discussion might be related to your question: https://github.com/awslabs/autogluon/discussions/1233

**Reply by Innixma** [reply]
Yes, assuming the images are 2D the same logic should hold as in #1233 where you can first convert the images to png and then use AutoGluon.

**Reply by Enderfga** [reply]
Thank you all for your replies!
